components/admin_console/admin_definition.jsx

```diff
@@ -5367,9 +5655,21 @@ const AdminDefinition = {
                         label: t('admin.service.enableBotTitle'),
                         label_default: 'Enable Bot Account Creation: ',
                         help_text: t('admin.service.enableBotAccountCreation'),
-                        help_text_default: 'When true, System Admins can create bot accounts for integrations in [Integrations > Bot Accounts]({siteURL}/_redirect/integrations/bots). Bot accounts are similar to user accounts except they cannot be used to log in. See [documentation](https://mattermost.com/pl/default-bot-accounts) to learn more.',
+                        help_text_default: 'When true, System Admins can create bot accounts for integrations in [Integrations > Bot Accounts]({siteURL}/_redirect/integrations/bots). Bot accounts are similar to user accounts except they cannot be used to log in. See <link>documentation</link> to learn more.',
```

Comment:
Hi, I was'tn sure I could include the {siteURL} variable in the value.
Something like this would work?
linkIntegrations: (msg) => (
                                <a
                                    href='{siteURL}/_redirect/integrations/bots'
                                    referrer='noreferrer'
                                    target='_blank'
                                    rel='noreferrer'
                                >
                                    {msg}
                                </a>
                            ),